src/extension.ts

```diff
@@ -5,29 +5,21 @@ import * as path from 'path';
 import * as fs from 'fs';
 import { exec } from 'child_process';
 
-let statusBarItem: vscode.StatusBarItem;
 let debugPanel: vscode.WebviewPanel | undefined;
 
 // This method is called when your extension is activated
 // Your extension is activated the very first time the command is executed
 export function activate(context: vscode.ExtensionContext) {
+	// Update context based on current editor
+    updateEhrqlContext();
 
-	// Create a status bar item that runs the display command
-    statusBarItem = vscode.window.createStatusBarItem(vscode.StatusBarAlignment.Right, 100);
-    statusBarItem.command = 'ehrql.debug';
-	statusBarItem.text = "Debug ehrQL";
-	statusBarItem.backgroundColor = new vscode.ThemeColor('statusBarItem.warningBackground');
-
-
-	// Implement the display command defined in the package.json file
+	// Implement the debug command defined in the package.json file
 	const disposable = vscode.commands.registerCommand('ehrql.debug', () => {
```

Comment:
The indenting goes a bit wrong here. Not for now, but maybe at some point we add a prettier config to enforce things like this.

Reply:
This is weird; the indenting is showing up in github differently to how it looks for me locally. I'm going to ignore it for now.